src/Microsoft.DotNet.Interactive.PowerShell/AzureShellConnection.cs

```diff
@@ -214,6 +245,13 @@ internal async Task ConnectAndInitializeAzShell(int terminalWidth, int terminalH
                 await RefreshToken().ConfigureAwait(false);
             }
 
+            var userSettings = await ReadCloudShellUserSettings().ConfigureAwait(false);
+            if (userSettings?.properties == null || userSettings.properties.storageProfile == null)
+            {
+                Console.WriteLine("It seems you haven't setup your cloud shell account yet. Navigate to https://shell.azure.com to complete account setup.");
```

Comment:
Should this be on `Console.Error`?

Reply:
Right, I think the output using `Console.Error` looks better.

Reply:
The screenshot in PR description was updated to show what it looks like with `Console.Error`.